js/scripts.js

```diff
@@ -0,0 +1,15 @@
+var size = prompt('podaj wysokosc')
+
+function rysujChoinke (size) {
```

Comment:
variables, functions and everything what is connected with code language we should name in english - so please remove our beauty language from here :)